### ISSUE TITLE
Separate different discord versions (stable, canary, etc) into their own replugged folders

### ISSUE THREAD

**Issue by Tracenji** [replugged-org/replugged#407]
### Describe the feature request

separate all the discord versions into their own replugged folder, allowing different plugins, themes, and quickcss on different discord versions (stable, ptb, canary, development)
as of now, at least on windows, they all share the same plugins, themes, etc, they even share which plugins/themes are enabled

**folder structure examples**
replugged 
replugged-canary
etc

or
replugged/stable
replugged/canary
etc

### Why should this be added?

it should be added as it would be useful to have the ability to use different plugins, themes, etc on the different discord release branches 

### Please confirm the following

- [X] This is a feature request for Replugged itself, not a feature request for a plugin or theme
- [X] [I searched the issues](https://github.com/replugged-org/replugged/issues) to make sure this feature has not been requested before.

**Comment by thewilloftheshadow** [comment]
If implemented, I would want a way to disable this, because I see it as a feature and like having my things synced across clients

**Comment by Tracenji** [comment]
yes, i can see that, either as a option at install or in the settings
i would not care if all in one folder was the default, i just want the option to have them separate
